integreat_cms/static/src/css/style.scss

```diff
@@ -227,7 +227,8 @@ select,
 [type="radio"] {
     @apply rounded text-blue-500 mr-2 align-text-top;
 
-    &:disabled {
+    &:disabled,
+    &:read-only {
```

Comment:
That's strange. I'm also on the latest FF and everything has been working. 

> I don't know what's the best workaround in this case, maybe we have to duplicate the field as hidden field to send its value even if it is disabled?

Yeah, that's probably the only way then :/
Or enabling the checkbox right before the form submit... 😅

Reply:
sure, good idea!

Reply:
Ah, but this would then cause the problem can end up in a "dead lock" when the checkbox is checked (but "disabled"), and the hix value is too low?
Because the page form cannot be saved any longer (because the warning shows), and the checkbox cannot be unchecked?
Or is the idea then to click "hix ignore", which should not only disable but also uncheck the MT checkbox?